docs/diagrams/AddClaimSequenceDiagram_Model.puml

```diff
@@ -0,0 +1,62 @@
+@startuml
+!include style.puml
+
+box Model MODEL_COLOR_T1
+participant ":ModelManager" as ModelManager MODEL_COLOR
```

Comment:
This can be broken into smaller sequence diagrams as there are multiple actions within this diagram